msi/GitChooseEnvironmentDlg.wxs

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
In general, there are two types of people that install software. The ones that just want the software and don't care that much and the others want control over everything. In most software the first group (those that don't care) vastly out number the second group. With developers, I expect we get a more even split.

Even then picking correct defaults to make Git for Windows work well on Windows is a good thing and letting those that need more can tweak to their desire.

And yes, my last comment is that WiX v4 can be improved/simplified/changed if we find things that should be done. That's why I wanted to go with WiX v4 here.

PS: for your testing, if you're not testing UI, I high recommend `/qb` on the command-line.